Arcs/Arcs.cpp

```diff
@@ -97,9 +97,11 @@ static const char USAGE_MESSAGE[] =
             "       --dist_median       use median distance in ABySS dist.gv [default]\n"
             "       --dist_upper        use upper bound distance in ABySS dist.gv\n"
             "       --dist_tsv=FILE     write min/max distance estimates to FILE\n"
-            "       --samples_tsv=FILE  write intra-contig distance/barcode samples to FILE\n";
+            "       --samples_tsv=FILE  write intra-contig distance/barcode samples to FILE\n"
+            "   -P, --pair              output scaffolds pairing tsv with number of evidences "
```

Comment:
```suggestion
            "   -P, --pair              output scaffolds pairing TSV with number of barcode links "
```
Is this accurate? I'm still not clear with how this differs from the normal .gv output

Reply:
gv is post threshold. I want it pre threshold in case I have to try the variable threshold approach.

Reply:
What threshold do you mean?

Reply:
The p value threshold (0.05). I want a file with barcode links of all four orientation for each scaffold pair, not just the best one

Reply:
Ok.. So what about 
```suggestion
            "   -P, --pair              output scaffolds pairing TSV with number of barcode links (no p-value threshold)"
```
or something similar?